gui/slick/interfaces/default/home.mako

```diff
@@ -814,6 +818,15 @@ $(document).ready(function(){
             </script>
         </td>
 
+        <% show_size = 0 %>
+        % try:
+            <% show_size = sickbeard.helpers.get_size(curShow.location) %>
+        % except sickbeard.exceptions.ShowDirNotFoundException:
+            <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        % endtry
+
```

Comment:
The exception if thrown [here](https://github.com/SiCKRAGETV/SickRage/blob/master/sickbeard/tv.py#L189), that's what the check is for.